packages/admin/admin-theme/src/index.ts

```diff
@@ -1,5 +1,6 @@
-export { bluePalette, errorPalette, greenPalette, infoPalette, neutrals, successPalette } from "./colors";
+export { breakpointsOptions, breakpointValues } from "./breakpointsOptions";
+export { errorPalette, greyPalette, infoPalette, primaryPalette, successPalette, warningPalette } from "./colors";
 export { createCometTheme } from "./createCometTheme";
 export { paletteOptions } from "./paletteOptions";
 export { shadows } from "./shadows";
-export { fontWeights, typographyOptions } from "./typographyOptions";
+export { createTypographyOptions, fontWeights } from "./typographyOptions";
```

Comment:
I'm not sure if any of these exports have any real value. We could think about omitting all values except for `createCometTheme()` as

- we want people to use the theme (instead of using a color palette directly) because colors could be overridden in the project
- in `createCometTheme()` we do a deep merge, so all non-overridden values are are preserved

Reply:
True, can't think of any situation where these values would be needed, other during the theme-creation itself.